i18n/en-US.yml

```diff
@@ -78,13 +78,13 @@ common:
     tuesday: Tuesday
     wednesday: Wednesday
   daysOfWeekCompact:
-    friday: Fri.
-    monday: Mon.
-    saturday: Sat.
-    sunday: Sun.
-    thursday: Thu.
-    tuesday: Tue.
-    wednesday: Wed.
+    friday: F
+    monday: M
+    saturday: S
+    sunday: Su
+    thursday: Th
+    tuesday: T
+    wednesday: W
```

Comment:
<img width="799" alt="Screenshot 2024-04-11 at 1 05 27 PM" src="https://github.com/opentripplanner/otp-react-redux/assets/86619099/ee6ab1d3-89ac-41f6-baea-865a1c419f6d">
This... isn't great. Is it possible to rework this component to have the same look and feel as the cool new circle ui? or are we thinking of doing that in a second PR

Reply:
I agree, we should use the new styles... but in a subsequent PR.